build.gradle

```diff
@@ -54,6 +57,25 @@ allprojects {
     configFile = new File(rootDir, "checkstyle/checkstyle.xml")
   }
 
+  task createConfigDocDir() {
+    outputs.dir configDocDir
+    file(configDocDir).mkdirs()
+  }
+
+  task createConfigDocs( dependsOn : ['createConfigDocDir', 'compileJava'], type : JavaExec) {
+    classpath sourceSets.main.runtimeClasspath
+    main = 'com.linkedin.kmf.common.ConfigDocumentationGenerator'
+    args = [configDocDir,
+            'com.linkedin.kmf.services.configs.ConsumeServiceConfig',
+            'com.linkedin.kmf.services.configs.DefaultMetricsReporterServiceConfig',
+            'com.linkedin.kmf.services.configs.JettyServiceConfig',
+            'com.linkedin.kmf.services.configs.ProduceServiceConfig',
+            'com.linkedin.kmf.services.configs.TopicManagementServiceConfig',
+            'com.linkedin.kmf.apps.configs.MirrorPipelineMonitorConfig']
```

Comment:
Can you rebase the patch and change MirrorPipelineMonitorConfig to MultiClusterMonitorConfig?